client/src/pages/NeedHelp.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from "react";
 import { withRouter } from "react-router-dom";
-
+import {Link} from "react-router-dom";
```

Comment:
@robinv85 Should we add a lint rule that enforces object spacing? Maybe we can look into [this](https://www.google.com/search?q=eslint+object+spacing&oq=eslint+object+spaing&aqs=chrome.1.69i57j0l6.3858j0j1&sourceid=chrome&ie=UTF-8)?